Source/WebCore/platform/MediaStrategy.cpp

```diff
@@ -83,4 +87,27 @@ void MediaStrategy::addMockMediaSourceEngine()
 }
 #endif
 
+#if ENABLE(VIDEO)
+RefPtr<AudioVideoRenderer> MediaStrategy::createAudioVideoRenderer(WTF::LoggerHelper* loggerHelper) const
```

Comment:
Why not LoggerHelp&?

Reply:
so it can be null for the windows build that don't have logging enabled.
I didn't want to have to constructor, one with release_log enabled taking two arguments, and one without release_log